src/Config/CS_SDK.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
-    <Platforms>NET60_Windows;NET60_Linux;</Platforms>
+    <Platforms>AnyCPU;NET60_Linux;</Platforms>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">NET60_Windows</Platform>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
```

Comment:
hmm, so what is the point of these conditions now? Just to avoid linux? Can we remove most of these?

Reply:
Yes, let me clean up that file.